pkg/resourceinterpreter/default/thirdparty/thirdparty_test.go

```diff
@@ -102,11 +108,11 @@ type TestStructure struct {
 }
 
 type IndividualTest struct {
-	DesiredInputPath  string `yaml:"desiredInputPath,omitempty"`
-	ObservedInputPath string `yaml:"observedInputPath,omitempty"`
-	StatusInputPath   string `yaml:"statusInputPath,omitempty"`
-	DesiredReplica    int64  `yaml:"desiredReplicas,omitempty"`
```

Comment:
This parameter is the replica parameter specified for revise and does not need to be deleted.